articles/getting_to_know/howto/HowTo_Install_Preview_Release.md

```diff
@@ -0,0 +1,88 @@
+---
+title: How to install a preview release of MonoGame from NuGet
+description: This topic describes how use a preview release of MonoGame in a new or existing project.
+---
+
+## Overview
+
+MonoGame provides preview deployments as testing releases prior to the full release of a new version, this gives developers early access to new features before they are made officially available.
```

Comment:
deployments `for` testing. Full stop after version, otherwise the sentence is a bit long.